tests/e2e/test_rosacli_cluster.go

```diff
@@ -328,9 +328,149 @@ var _ = Describe("Edit cluster",
+				_, err = clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=aaa",
+					"-y",
+				)
+				Expect(err).To(HaveOccurred())
+				textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+				Expect(textData).Should(ContainSubstring(
+				`Error: invalid argument "aaa" for "--enable-delete-protection" flag: strconv.ParseBool: parsing "aaa": invalid syntax`))
+
+				_, err = clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=",
+					"-y",
+				)
+				Expect(err).To(HaveOccurred())
+				textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+				Expect(textData).Should(ContainSubstring(
+				`Error: invalid argument "" for "--enable-delete-protection" flag: strconv.ParseBool: parsing "": invalid syntax`))
+
+				By("Attempt to delete cluster with delete protection enabled")
+				_, err = clusterService.DeleteCluster(clusterID, "-y")
+				Expect(err).To(HaveOccurred())
+				textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
+				Expect(textData).Should(ContainSubstring(
+				`Delete-protection has been activated on this cluster and it cannot be deleted until delete-protection is disabled`))
+
+				By("Disable delete protection on the cluster")
+				deleteProtection = "Disabled"
+				_, err = clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=false",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Check the disable result from cluster description")
+				output, err = clusterService.DescribeCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+
+				clusterDetail, err = clusterService.ReflectClusterDescription(output)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(clusterDetail.EnableDeleteProtection).To(Equal(deleteProtection))
+			})
+
+		It("can allow sts cluster installation with compatible policies - [id:45161]",
+			labels.High, labels.Runtime.Day1Supplemental,
+			labels.Runtime.Day2, // day2 is still labeled to make sure it can be covered in the CI. we need to remove it with day1 supplemental job setup
+			func() {
+				By("Check the cluster is STS cluster or skip")
+				isSTSCluster, err := clusterService.IsSTSCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				isHostedCluster, err := clusterService.IsHostedCPCluster(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				if !isSTSCluster || isHostedCluster {
+					SkipTestOnFeature("policy")
+				}
+
+				clusterName := "cluster-45161"
+				operatorPrefix := "cluster-45161-asdf"
+				isHostedCP, err := clusterService.IsHostedCPCluster(clusterID)
+				Expect(err).To(BeNil())
+
+				By("Create cluster with one Y-1 version")
+				ocmResourceService := rosaClient.OCMResource
+				versionService := rosaClient.Version
+				accountRoleList, _, err := ocmResourceService.ListAccountRole()
+				Expect(err).To(BeNil())
+				rosalCommand, err := config.RetrieveClusterCreationCommand(ciConfig.Test.CreateCommandFile)
+				Expect(err).To(BeNil())
```

Comment:
This is not existing anymore if it's day1 supplemental case. The case itself should can be run independently,